react/src/js/components/Consonant/Infobit/Type/Link.jsx

```diff
@@ -31,12 +33,14 @@ const Link = ({
     href,
     linkHint,
     text,
+    title,
```

Comment:
Recommend to rename to `linkTitle` as `title` is used below as an html attribute.